frontend/src/app/projects/models/backup-settings/job-run-overview/job-run-overview.component.ts

```diff
@@ -30,26 +43,91 @@ export class JobRunOverviewComponent implements OnInit {
 
   ngOnInit() {
     combineLatest([
-      this.projectService.project.pipe(filter(Boolean)),
-      this.modelService.model.pipe(filter(Boolean)),
-      this.pipelineService.pipeline.pipe(filter(Boolean)),
+      this.projectService.project.pipe(
+        filter(Boolean),
+        map((project) => project.slug)
+      ),
+      this.modelService.model.pipe(
+        filter(Boolean),
+        map((model) => model.slug)
+      ),
+      this.pipelineService.pipeline.pipe(
+        filter(Boolean),
+        map((pipeline) => pipeline.id)
+      ),
     ])
       .pipe(
         untilDestroyed(this),
-        switchMap(([project, model, pipeline]) =>
+        switchMap(([projectSlug, modelSlug, pipelineID]) =>
           this.pipelineRunService.loadPipelineRuns(
-            project.slug,
-            model.slug,
-            pipeline.id
+            projectSlug,
+            modelSlug,
+            pipelineID,
+            1,
+            this.pageSize
           )
         )
       )
       .subscribe();
   }
 
+  ngAfterViewInit(): void {
```

Comment:
Similar to the `ngOnInit` I would change this to the following to increase readability. In addition, there was an `untilDestroy(this)` missing.

```typescript
combineLatest([
      this.projectService.project.pipe(filter(Boolean)),
      this.modelService.model.pipe(filter(Boolean)),
      this.pipelineService.pipeline.pipe(filter(Boolean)),
    ])
      .pipe(untilDestroyed(this))
      .subscribe(([project, model, pipeline]) => {
        this.observeVisibleJobs(project.slug, model.slug, pipeline.id);
      });
```